reference/tools.rst

```diff
@@ -1622,3 +1622,73 @@ Converts Conan style architecture into Android NDK style architecture.
 
 Parameters:
     - **arch** (Required): Arch to perform the conversion. Usually this would be ``self.settings.arch``.
+
+.. _tools.check_min_cppstd:
+
+tools.check_min_cppstd()
+-------------------------------
+
+.. code-block:: python
+
+    def check_min_cppstd(conanfile, cppstd, gnu_extensions=False)
+
+Validate the current cppstd from settings or compiler, if it is supported by the required cppstd version.
+It raises a ``InvalidConfiguration`` when is not supported.
```

Comment:
That exception doesn't exist. Is ``ConanInvalidConfiguration``. Link to https://docs.conan.io/en/latest/mastering/conditional.html

Reply:
done